syntaxes/cfg.tmLanguage.json

```diff
@@ -1,165 +1,152 @@
-          "captures": {
-            "1": {
-              "name": "string.quoted.double.cfg"
-            },
-            "3": {
-              "name": "string.quoted.double.cfg"
-            },
-            "2": {
-              "name": "support.class"
-            }
-          }
-        }
-      ]
-    },
-    "actions": {
-      "patterns": [
-        {
-          "name": "keyword.other.cfg",
-          "match":
-            "(?i)([\\+|-]attack\\d?|[\\+|-]jump|[\\+|-]duck|[\\+|-]forward|[\\+|-]back|[\\+|-]moveleft|[\\+|-]moveright|[\\+|-]moveup|[\\+|-]movedown|[\\+|-]reload|[\\+|-]inspect|[\\+|-]use_action_slot_item|[\\+|-]voicerecord|[\\+|-]taunt|[\\+|-]lookup|[\\+|-]lookdown|[\\+|-]showscores|[\\+|-]strafe|[\\+|-]klook|[\\+|-]showroundinfo|[\\+|-]camdistance|[\\+|-]camin|[\\+|-]camout|[\\+|-]cammousemove|[\\+|-]campitchdown|[\\+|-]campitchup|[\\+|-]camyawleft|[\\+|-]camyawright)\\b"
-        },
-        {
-          "name": "keyword.other.cfg",
-          "match":
-            "(?i)\\b(say|say_team|say_party|lastinv|impulse|build|destroy|eureka_teleport|disguise|load_itempreset|voicemenu|voice_menu_\\d|join_class|vote|changeclass|changeteam|open_charinfo_\\w+|show_quest_log|disguiseteam|cancelselect|pause|quit|toggleconsole|screenshot|save_replay|askconnect_accept|abuse_report_queue|name|lastdisguise|dropitem|centerview|showmapinfo|inspect|invnext|invprev|wait|taunt_by_name|slot\\d{1,2})\\b"
-        },
-        {
-          "name": "keyword.other.cfg",
-          "match": "(?i)\\b(spec_\\w+)\\b"
-        }
-      ]
-    },
-    "settings": {
-      "patterns": [
-        {
-          "name": "keyword.control.cfg",
-          "match":
-            "(?i)\\b(exec|alias|bind|bind_mac|bindtoggle|toggle|unbind|unbindall|volume|crosshair|sensitivity|rate|map|deathmatch|tracer_extra|in_usekeyboardsampletime|joystick|fast_fogvolume|incrementvar|joy_\\w+|fov_\\w+|viewmodel\\w+|m_\\w+|map_\\w+|r_\\w+|net_\\w+|cl_\\w+|tf_\\w+|sv_\\w+|con_\\w+|obj_\\w+|hud_\\w+|sixense_\\w+|windows_\\w+|snd_\\w+|mat_\\w+|mp_\\w+|flex_\\w+|lod_\\w+|rope_\\w+|glow_\\w+|demo_\\w+|func_\\w+|violence_\\w+|dsp_\\w+|props_\\w+|ragdoll_\\w+|ai_\\w+|g_\\w+|cam_\\w+|sb_\\w+|mod_\\w+|nav_\\w+|filesystem_\\w+|budget__\\w+|c_\\w+|cache_\\w+|replay_\\w+|vr_\\w+|pyro_\\w+|nb_\\w+|ent_\\w+|fire_\\w+|fog_\\w+|ifm_\\w+|ds_\\w+|host_\\w+|cc_\\w+|g_\\w+|gameui_\\w+|full|blink_duration|bloodspray|bot|bot_\\w+)\\b"
-        }
-      ]
-    },
-    "unknown": {
-      "patterns": [
-        {
-          "name": "invalid.deprecated",
-          "match": "\\b(show_matchmaking|saveme|god)\\b"
-        },
-        {
-          "name": "invalid.deprecated",
-          "match": "([\\+|-]mlook|[\\+|-]alt[1|2])"
-        }
-      ]
-    }
-  }
-}
+	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
```

Comment:
It looks like this diff is so massive because it's swapping from 2-spaces to 4-spaces? I'm not too picky about the style choice, but it would be easier to read the diff if it wasn't treated as a wholesale swap. Again, if you just wanna take over I can ignore the details and just let you take over.

Reply:
Yeah, I had just formatted the code here, I do want to take over since I feel like I'm going to be able to contribute a lot more to it given you moved on from tf2 yourself.